### DISCUSSION TITLE
Alluvial Diagram

### DISCUSSION THREAD

**Discussion by dejudicibus** [rawgraphs/rawgraphs-app#344] [Q&A]
Is it possible to add an option to specify the position of the labels with respect the nodes, that is

```
LEFT         label |-----label |----- label |
RIGHT        | label-----| label-----| label
EXTERNAL     label |-----------------| label
```

External is an important feature when you have two groups of nodes because labels do not overwrite links, and have a much better visibility.

Thank you in advance.

**Comment by Ourymali** [comment]
Sorry
